server/api/test/test.js

```diff
@@ -6,23 +6,23 @@ var expect = require( 'chai' ).expect,
 	request = require( 'superagent' ),
 	supertest = require( 'supertest' );
 
-/**
- * Internal dependencies
- */
-var config = require( 'config' ),
-	app = require( '../' )(),
-	localRequest = supertest( app );
-
 if ( process.env.NODE_ENV === 'desktop' ) {
 	describe( 'api', function() {
-		var sandbox = sinon.sandbox.create();
+		const sandbox = sinon.sandbox.create();
+		let app, config, localRequest;
+
+		before( () => {
```

Comment:
I prefer to be on a safe side and require all stuff here. I opened follow up to refactor this very `desktop` dependent test.